### ISSUE TITLE
KolourPaint can't save images

### ISSUE THREAD

**Issue by KapiX** [haikuports/haikuports#3274]
Wherever you try to save the file, it will say there was an error.

**Comment by diversys** [comment]
Run `kbuildsycoca5 --global` to fix it.

**Comment by KapiX** [comment]
That fixed it.

**Comment by diversys** [comment]
We should run this command on kservice package activation.

**Comment by threedeyes** [comment]
> We should run this command on kservice package activation.
No. We need do this in each kde packages, which contains with mime types data.
